dcos/subcommand.py

```diff
@@ -543,20 +543,20 @@ def _install_with_pip(
             for line in requirements:
                 print(line, file=requirements_file)
 
-        cmd = [
-            os.path.join(env_directory, BIN_DIRECTORY, 'pip'),
-            'install',
-            '--requirement',
-            requirement_path,
-        ]
-
-        if _execute_command(cmd)[2] != 0:
-            # We should remove the directory that we just created
-            if new_package_dir:
-                shutil.rmtree(env_directory)
-
-            raise _generic_error(package_name)
-
+        with util.set_env('__PYVENV_LAUNCHER__', None):
```

Comment:
Let's write a comment here as to why we are doing this. In the future we can put the `setuptools` GitHub issue when we have it.